Godeps/Godeps.json

```diff
@@ -8,10 +8,12 @@
 	"Deps": [
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
+			"Comment": "null-5",
```

Comment:
I didn't change the copy in staging. hack/verify doesn't check repos, so we should be ok.